nucypher/blockchain/eth/actors.py

```diff
@@ -851,7 +854,6 @@ def to_dict(self) -> dict:
     @property
     def is_staking(self) -> bool:
         """Checks if this Staker currently has active stakes / locked tokens."""
-        self.stakes.refresh()
```

Comment:
I'm trying to make abstraction more clear. `Staker` methods assume that `stakes` are already refreshed - because refreshing is heavy and should be controlled outside